pandas/core/generic.py

```diff
@@ -7366,16 +7366,32 @@ def resample(self, rule, how=None, axis=0, fill_method=None, closed=None,
                  label=None, convention='start', kind=None, loffset=None,
                  limit=None, base=0, on=None, level=None):
         """
+        Resample time-series data.
+
         Convenience method for frequency conversion and resampling of time
-        series.  Object must have a datetime-like index (DatetimeIndex,
-        PeriodIndex, or TimedeltaIndex), or pass datetime-like values
-        to the on or level keyword.
+        series.  Object must have a datetime-like index (``DatetimeIndex``,
+        ``PeriodIndex``, or ``TimedeltaIndex``), or pass datetime-like values
+        to the ``on`` or ``level`` keyword.
 
         Parameters
         ----------
         rule : string
```

Comment:
Can you use the Python types (`str` instead of `string`...)

Reply:
Thanks, done. The module is full of `string` by the way. Should I create another issue for that? Do these values `str`, `int` get picked up by Sphinx?

Reply:
Yeah, atomic improvements around each docstring make total sense. Pushing the amendments now.